bin.js

```diff
@@ -45,13 +50,21 @@ function getConfig(file) {
     // point to the particular config
     configFile: file,
   })
-  var config = cliEngine.getConfigForFile(file)
-  return config
+  return cliEngine.getConfigForFile(file)
 }
 
-function getRules() {
-  var configFile = getConfigFile()
-  var config = getConfig(configFile)
+function getCurrentRules() {
```

Comment:
You invoke the method with `config` given as an argument in line 15 but use the global `config` later in the function's body. Would you please add `config` as an argument?

Same with `getPluginRules` in line 61.